docs/source/en/using-diffusers/inpaint.md

```diff
@@ -216,20 +216,20 @@ make_image_grid([init_image, mask_image, image], rows=1, cols=3)
 
 ## Non-inpaint specific checkpoints
 
-So far, this guide has used inpaint specific checkpoints such as [runwayml/stable-diffusion-inpainting](https://huggingface.co/runwayml/stable-diffusion-inpainting). But you can also use regular checkpoints like [runwayml/stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5). Let's compare the results of the two checkpoints.
+So far, this guide has used inpaint specific checkpoints such as [runwayml/stable-diffusion-inpainting](https://huggingface.co/runwayml/stable-diffusion-inpainting). But you can also use regular checkpoints like [Lykon/dreamshaper-8](https://huggingface.co/Lykon/dreamshaper-8). Let's compare the results of the two checkpoints.
```

Comment:
Let's also update the inpainting checkpoint

```suggestion
So far, this guide has used inpaint specific checkpoints such as [Lykon/dreamshaper-8-inpainting](Lykon/dreamshaper-8-inpainting). But you can also use regular checkpoints like [Lykon/dreamshaper-8](https://huggingface.co/Lykon/dreamshaper-8). Let's compare the results of the two checkpoints.
```

Reply:
hi, @stevhliu. Fixed

Reply:
they also exist here https://huggingface.co/stable-diffusion-v1-5/stable-diffusion-v1-5 FWIW

Reply:
for future reproducibility, cc @apolinario

Reply:
Actually, if you don't mind @suzukimain, can you replace `Lykon/dreamshaper-8` with the `stable-diffusion-v1-5/stable-diffusion-v1-5` checkpoints?

Reply:
hello, @stevhliu.  `Lykon/dreamshaper-8` was replaced by `stable-diffusion-v1-5/stable-diffusion-v1-5`.
Also, please check `docs/source/en/using-diffusers/inpaint.md`

https://github.com/huggingface/diffusers/blob/302720ae3da8da74f3f150537035ce6057dee75f/docs/source/en/using-diffusers/inpaint.md?plain=1#L220